modules/apps/headless-cms/layout-rest-api/src/main/java/com/liferay/layout/rest/model/WebPage.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.layout.rest.model;
+
+import aQute.bnd.annotation.ProviderType;
+
+import com.liferay.apio.architect.language.Language;
+
+import java.util.Date;
+
+/**
+ * Represents a web page.
+ *
+ * <p>
+ * Conforms to the <a href="http://schema.org/WebPage">WebPage </a> type from
+ * schema.org.
+ * </p>
+ *
+ * @author Pavel Savinov
+ */
+@ProviderType
+public interface WebPage {
```

Comment:
@ahdezma Could you propose Pavel the recommended way to avoid having to create this custom interfaces with the newest version of Apio?

Pavel, I'm ok with leaving it as it is as a first step, but Alex can propose a simpler implementation that should be followed for this and future APIs